package.json

```diff
@@ -10,8 +10,8 @@
   "author": "Apollo <packages@apollographql.com>",
   "license": "MIT",
   "engines": {
-    "node": ">=12.13.0",
-    "npm": ">=7 <9"
+    "node": ">=14",
+    "npm": ">7 <=9"
```

Comment:
Do we truly need an upper bound here? (Though given that this is a monorepo package.json rather than a published package it doesn't matter that much)

Reply:
Nah, I'm on board with dropping it. The fact that it's only a development concern within this monorepo makes it hardly an issue.